src/cloud/components/organisms/CommentManager.tsx

```diff
@@ -192,6 +192,7 @@ function CommentManager({
 
   return (
     <Container>
+      <ContextMenuClose />
```

Comment:
I don't actually think this should be here. The right side draw and its behaviour should maybe be handled in a wrapper or something? The CommentManager itself shouldn't care what container it is in.